<launch>

  <node
      name="ros_vicon_offset"
      pkg="ros_pybullet_interface"
      type="vicon_tf_offset.py"
      output="screen">

    <param
        name="object_name"
        value="hanging_obj_plate"
        type="str"/>

    <param
        name="offset"
        value="0.0 -0.0035 -0.2093 0.0 0.0 0.0"
        type="str"/>

  </node>

  <node
      name="ros_vicon_offset_yang_base"
      pkg="ros_pybullet_interface"
      type="vicon_tf_offset.py"
      output="screen">

    <param
        name="object_name"
        value="yang_frame"
        type="str"/>

    <param
        name="offset"
        value="-0.122 0.43133 -0.01 0.0 0.0 0.0"
        type="str"/>

  </node>

  <node
      name="ros_vicon_offset_yin_base"
      pkg="ros_pybullet_interface"
      type="vicon_tf_offset.py"
      output="screen">

    <param
        name="object_name"
        value="yin_frame"
        type="str"/>

    <param
        name="offset"
        value="0.12433 0.429 -0.01 180.0 0.0 0.0"
        type="str"/>

  </node>


</launch>
